### DISCUSSION TITLE
port to W5500-EVB-Pico (RP2040 + W5500)

### DISCUSSION THREAD

**Discussion by vikmad** [vintlabs/fauxmoESP#240] [Ideas]
Would it be possible?

BENEFIT:
+ low price and availability
+ LAN port instead of Wifi

W5500-EVB-Pico is a microcontroller evaluation board based on the Raspberry Pi RP2040 and fully hardwired TCP/IP controller W5500 – and basically works the same as Raspberry Pi Pico board but with additional Ethernet via W5500.

**Comment by pvint** [comment]
I recently accepted a PR that should make this work (see #234)!

It's not in the released version, but the current master branch includes it. Note that I haven't personally tested it.
